readthedocs/doc_builder/python_environments.py

```diff
@@ -317,8 +416,15 @@ def setup_base(self):
         cli_args.append(
             self.venv_path(),
         )
+
+        # TODO: make ``self.config.python_interpreter`` return the correct value
+        if self.project.has_feature(Feature.USE_NEW_DOCKER_IMAGES_STRUCTURE):
+            python_interpreter = 'python'
```

Comment:
Is this because we're not parsing out the correct interpreter without the config file changes?

Reply:
Yes. When the changes to the config file are implemented, we will always want just the executable name without the version since it will be the only version enabled globally (`asdf global python X.y.z`). So, `python` will always point to the right version

Reply:
I'd expect `self.config.python_interpreter` to return always the correct interpreter depending on the case, based on the other configs